modules/core/src/v2/coins/eth.ts

```diff
@@ -283,12 +283,14 @@ interface RecoverTokenTransaction {
 
 export class Eth extends BaseCoin {
   static hopTransactionSalt = 'bitgoHopAddressRequestSalt';
+  protected sendMethodName: string;
```

Comment:
Could this be typed a bit tighter?
```suggestion
  protected readonly sendMethodName: 'sendMultiSig' | 'sendMultiSigToken';
```

Reply:
I think this will have to be typed like this because if I will assign it as readonly, then I will not be able to assign its value in the constructors, right?
`protected sendMethodName: 'sendMultiSig' | 'sendMultiSigToken';`

Reply:
I have updated the type. Can you please once review?